network-api/networkapi/wagtailpages/templates/wagtailpages/fragments/entry_cards_item_loop.html

```diff
@@ -1,11 +1,11 @@
 {% for entry in entries %}
-  <li class="tw-m-0 tw-p-0">
-    {% with type=entry.specific_class.get_verbose_name|lower %}
+{% with type=entry.specific_class.get_verbose_name|lower %}
+  <li class="tw-m-0 tw-p-0 {% if type == "blog page" %} tw-pt-5 {% endif %}">
```

Comment:
Adding padding to the top of the blog cards, that way they will show the divider from the tailwind divide class